modules/n1ql/pages/n1ql-language-reference/typefun.adoc

```diff
@@ -127,6 +139,14 @@ SELECT ISBOOLEAN(true) AS `boolean`,
 
 === Description
 
+Type functions perform operations that check or convert expressions.
+
+=== Arguments
+
+expression:: [Required] The expression to check.
+
+=== Return Value
+
 Returns True if expression is a number, otherwise returns MISSING, NULL or false.
 
 === Examples
```

Comment:
The result of the example is still wrong. Look at the description of this function: we would expect that it will return `true` for `number`; `null` for `null`; and `false` for everything else.